src/db.rs

```diff
@@ -71,6 +71,59 @@ impl DB {
         Ok(())
     }
 
+    pub fn migrate<P: AsRef<Path>>(&mut self, path: P) -> Result<()> {
+        if !self.dirs.is_empty() {
+            return Err(anyhow!(
+                "To prevent conflicts, you can only migrate from z with an empty \
+                 zoxide database!"
+            ));
+        }
+
+        let z_db_file = File::open(path).with_context(|| anyhow!("could not open z db file"))?;
+        let reader = BufReader::new(z_db_file);
+
+        for (idx, read_line) in reader.lines().enumerate() {
+            let line_number = idx + 1;
+            let line = if let Ok(line) = read_line {
+                line
+            } else {
+                eprintln!("could not read line {}: {:?}", line_number, read_line);
+                continue;
+            };
+
+            let split_line = line.rsplitn(3, '|').collect::<Vec<&str>>();
+
+            match split_line.as_slice() {
+                [_, rank_str, path_str] => {
+                    // Set last_accessed to current time; otherwise, the rank will get scaled down,
```

Comment:
I don't think we should do this. IMO, the frecency _should_ get scaled down if the entry is old. Immediately after importing, if all the entries are old, all of them will be scaled equally, and there will be no problem. Once the user starts using `zoxide`, the new entries created will be prioritized, which is also intended behavior.

Reply:
Fair.